package.json

```diff
@@ -41,7 +41,7 @@
     "build:bundle-stats": "webpack --progress --bail --mode production --json > webpack-stats.json",
     "build:module_system": "tsc --project ./tsconfig.module_system.json && node ./lib/module_system/scripts/install.js",
     "dist": "scripts/package.sh",
-    "start": "concurrently --kill-others-on-fail --prefix \"{time} [{name}]\" -n res,element-js \"yarn start:res\" \"yarn start:js\"",
+    "start": "yarn build:module_system && concurrently --kill-others-on-fail --prefix \"{time} [{name}]\" -n res,element-js \"yarn start:res\" \"yarn start:js\"",
```

Comment:
This also needs the jitsi bit :(

Reply:
which jitsi bit? That should be picked up from the `res` stuff and always has

Reply:
oh yes, confusingly `start:res` isn't just the --start equiv of `build:res` - what a mess, my bad